source/includes/_zapier.md

```diff
@@ -10,19 +10,19 @@ NOTE: There is an additional monthly subscription cost through Zapier® The cost
 
 **Parameters:**
 
-With the LOCALiQ App you can get contacts and contacts interactions for ingestion into the CRM of your choice. The CRM should also have an app available on Zapier®.
+With the LOCALiQ App you can get contacts and contact interactions for ingestion into the CRM of your choice. The CRM must also have an app available on Zapier®.
 
-Contacts: This HIPAA Compliant endpoint supports integration with CRM products. It returns all contacts (first name, last name, email and/or phone number) generated by a campaign for an advertiser. The response is organized by campaigns with the contacts nested within. Please note that the response also includes the associated contact and its meta data.
+Contacts: This HIPAA Compliant endpoint supports integration with CRM products. It delivers contact data (first name, last name, email and/or phone number) generated for an advertiser in near real-time.
```

Comment:
Replace "This HIPAA Compliant endpoint" with "This trigger".  I don't think it's HIPAA compliant.  
Add "etc" to list of fields.

Reply:
Updated to "This trigger..." and added "etc" to list of fields.